content/2020-06-05-pycones-2020-c4p-en.md

```diff
@@ -0,0 +1,101 @@
+Title: PyConES 2020 Call for Papers is open!!!
+Date: 2020-06-15
+Category: PyConES
+Summary: We are pleased to announce that the C4P for PyConES 2020 is open! Now you can send us your proposals for the PyConES 2020 conference and this year we will have some important changes!
+
+<center><small>
+NOTE: Si hablas español, puedes [ir a la versión es ESPAÑOL]({filename}/2020-06-05-pycones-2020-c4p-es.md)
+</small></center>
+
+We are pleased to announce that the C4P for **PyConES 2020** *(Online version)* is open!
+
+Now you can send us your proposals for the PyConES 2020 conference and this year we will have some important changes!
+
+
+## Our Code of Conduct
+
+We want to have a welcoming and pleasant conference, with a harassment-free experience for everyone. PyConES 2020 has a mandatory [code of conduct](https://www.es.python.org/pages/codigo-de-conducta.html) for all people participating in the event.
+
+
+## Where and when:
```

Comment:
I recommend not ending any header with `:` or ending them all with `:`. I personally prefer not using `:` in the header.

Reply:
:+1: